src/functions.date.js

```diff
@@ -138,13 +143,18 @@ _.duration = $.extend(function($this, ms, terms = 1) {
 		
 		if (unitTime !== 0 || terms === 1) {
 			let unitProperPlurality = unitTime === 1 && unit !== "ms" ? unit.slice(0, -1) : unit;
-			timeArray.push(unitTime + " " + _.phrase($this, unitProperPlurality));
-			timeLeft -= unit === "ms" ? unitTime : unitTime * Mavo.Functions[unit]();
+			ret.push(unitTime + " " + _.phrase($this, unitProperPlurality));
+			timeLeft -= unitTime * (unit === "ms" ? 1 : Mavo.Functions[unit]());
 			unitTime = timeLeft;
 		}
 		
 	}
-	return timeArray;
+	
+	if (wantString) {
+      		return ret[0];
+    	}
+		
+    	return ret;
```

Comment:
Rogue spaces for indentation. Spaces are ok for alignment, but we use tabs for indentation (this is called "smart tabs"). I thought ESLint warned about that?